modules/gapi/include/opencv2/gapi/gtransform.hpp

```diff
@@ -0,0 +1,98 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2019 Intel Corporation
+
+#ifndef OPENCV_GAPI_GTRANSFORM_HPP
+#define OPENCV_GAPI_GTRANSFORM_HPP
+
+#include <functional>
+#include <type_traits>
+#include <utility>
+
+#include <opencv2/gapi/gcommon.hpp>
+#include <opencv2/gapi/util/util.hpp>
+#include <opencv2/gapi/garg.hpp>
+#include <opencv2/gapi/gtype_traits.hpp>
+#include <opencv2/gapi/util/compiler_hints.hpp>
+
+namespace cv
+{
+
+struct GAPI_EXPORTS GTransform
+{
+    using F = std::function<GArgs(const GArgs &)>;
+
+    const char *description;
+    F pattern;
+    F substitute;
+
+    explicit GTransform(const char *d, const F &p, const F &s) : description(d), pattern(p), substitute(s){};
```

Comment:
There is no need for `explicit` in this case

Reply:
Removed